site/_posts/2023-03-09-react-example-application.md

```diff
@@ -0,0 +1,247 @@
+content=
+"This article was written using an early look at the FusionAuth Hosted OAuth Service Provider Endpoints. The feature may be not yet available in the latest version of FusionAuth."
+%}
+
+### Clone the Repository
+
+To begin, clone the demo application from GitHub using the following command:
+
+```bash
+git clone https://github.com/FusionAuth/fusionauth-react-example-app-placeholder-replaceme
+```
+
+### Set up the FusionAuth application
+
+To set up the FusionAuth application, follow the steps below:
+
+```bash
+cd fusionauth-react-example-app-placeholder-replaceme
+docker compose up -d
+# Wait until FusionAuth is set up and available at http://localhost:9011
+cd server && cp example.env .env && npm install && npm run seed && cd ..
+cd client && cp example.env .env && npm install && cd ..
+```
+
+### Run the Application
+
+To run the application, follow the steps below:
+
+```bash
+cd server && npm run start
+```
+
+Open a new terminal and run the following command:
+
+```bash
+cd client && npm run start
+```
+
+The application is now running on `http://localhost:3000`.
+
+Use the following credentials to log in:
+- Email: `admin@example.com`
+- Password: `password`
+
+This will log you in as an agent. You can also log in as a user by using the following credentials:
+- Email: `richard@example.com`
+- Password: `password`
+
+### Set up the Fusionauth Instance Manually
+
+If you do not want to use Docker, you can set up the FusionAuth instance manually. To do so, follow the steps detailed in the [FusionAuth documentation](https://fusionauth.io/docs/v1/tech/installation-guide/fusionauth-app) and the [README.md](https://github.com/FusionAuth/fusionauth-react-example-app-placeholder-replaceme/blob/main/README.md) of the example application.
+
+## Using Hosted OAuth Service Provider Endpoints
+
+Authentication is hard. We want to make it easy.
+That is why we have added the necessary endpoints for the OAuth 2.0 login flow into the FusionAuth API. This means that you can use FusionAuth to authenticate users. This is a great way to get started with FusionAuth and to use it as a drop-in replacement for your existing OAuth 2.0 provider. So you can concentrate on providing the best experience for your users.
+
+{% include _callout-important.liquid
+content=
+"For limitations and prerequisites, see the [Hosted OAuth Service Provider Endpoints](https://fusionauth.io/xyz) documentation."
+%}
+
+{% include _image.liquid src="/assets/img/blogs/react-example-application/fusiondesk-fusionauth-login.png" alt="Login Screen of FusionAuth with the styling of FusionDesk" class="img-fluid" figure=false %}
+
+As you can see - if you inspect the backend of the example application - we do not provide any OAuth 2.0 endpoints. Instead, we use the FusionAuth API to authenticate users.
+
+When the user accesses the backend, we validate the `access_token` cookie. If the cookie is valid, we will return the tickets. If the cookie is not valid, we return a 401 error.
```

Comment:
Feels like you should talk about how a user logs in before you talk about how the app functionality is accessed.